docs/custom-flows/bot-sign-up-protection.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Bot sign-up protection on a custom flow
+description: Learn how to use the "Smart" bot sign-up protection on your custom flow.
+---
+
+# Smart bot sign-up protection
+
+When using the `Smart` option at **User & Authentication > Attack Protection > Bot sign-up protection** in Clerk Dashboard, it's important to include a specific element in your DOM to facilitate the rendering of the CAPTCHA widget. 
```

Comment:
> If you have enabled the Smart Bot Protection widget under **User & Authentication ....** in Dashboard, it's required to include  a specific element in your DOM, to enable rendering of the widget.
>
> Specifically, there should be a `<div id="clerk-captcha" >/` element by the time you call `signUp.create()`. This element acts as a placeholder onto which the widget will be rendered.
>
> If this element is not found, the SDK will transparently fall back to the Invisible widget, in order to avoid breaking your sign-up flow. If this happens, you should see a relevant error in your browser's console.
>
> The Invisible widget is more prone to false positives, therefore we strongly recommend that you fix this error by ensuring the `<div id="clerk-captcha" />` element exists in your DOM.